ltm/profiles.go

```diff
@@ -63,17 +47,16 @@ func (l *LTM) UploadFile(file, name string) error {
 		return apierror.New(apierror.ErrBadRequest, "invalid input", nil)
 	}
 
-	uploads, err := l.Service.UploadBytes([]byte(file), name)
+	_, err := l.Service.UploadBytes([]byte(file), name)
 	if err != nil {
+		// GOOGLEY FIXME
```

Comment:
👀

Reply:
probably make this 

```golang
if _, err := l.Service.UploadBytes()....;err != nil {
...
}
```

also get rid of the else since you're returning in the if